app/api/cruds/user_crud.py

```diff
@@ -0,0 +1,106 @@
+from django.contrib.auth.models import User
+from django.core.exceptions import ValidationError
+
+import graphene
+from graphene_django import DjangoObjectType
+from .utils import get_user, get_errors
```

Comment:
@andela-aoduola: As I explained once in my earlier review comments, the import should be in order: python standard libraries, django libraries, third-party libraries and custom libraries. Note blank space should be used to separate them.

So following that, there should be a blank space between line 5 and 6.